test/test-servers/express_scaffold_server.ts

```diff
@@ -15,26 +15,26 @@
  */
 // jscs:disable requireEarlyReturn
 
-var WARNING_HEADER = '\n!! -WARNING-';
-var EXCLAMATION_LN = '\n!!';
-var has = require('lodash.has');
-var express = require('express');
-var app = express();
-var errorHandler = require('../../src/index.js')({
+const WARNING_HEADER = '\n!! -WARNING-';
+const EXCLAMATION_LN = '\n!!';
+import has = require('lodash.has');
+import * as express from 'express';
+const app = express();
+const errorHandler = require('../../src/index.js')({
   onUncaughtException: 'report',
   key: process.env.STUBBED_API_KEY,
   projectId: process.env.STUBBED_PROJECT_NUM,
 });
-var bodyParser = require('body-parser');
+const bodyParser = require('body-parser');
 
 // eslint-disable-next-line no-console
-var log = console.log;
+const log = console.log;
 
 app.use(bodyParser.json());
 
 app.post('/testErrorHandling', function(req, res, next) {
   if (has(req.body, 'test') && req.body.test !== true) {
-    return next(new Error('Error on Express Regular Error POST Route'));
+    return next!(new Error('Error on Express Regular Error POST Route'));
```

Comment:
Seems unrelated. Why is this being changed as part of this PR?

Reply:
Yes.  With the change to use `imports`, type checking indicates that `next` could be `null`.  I used `!` to cause the test that uses this code to fail if `next` is null.